### DISCUSSION TITLE
Shows incorrect number of repositories on my profile

### DISCUSSION THREAD

**Discussion by gmoniava** [community/community#17109] [Other Feature Feedback, Questions, & Ideas]
I have one repository, but it seems to show 2 on profile? See the screenshot.
<img width="359" alt="Capture" src="https://user-images.githubusercontent.com/77232558/169696992-9c057b19-6ca1-4548-919e-57606ab0e432.PNG">

PS. I don't have private repos.

**Comment by NotchL1118** [comment]


**Reply by gmoniava** [reply]
No I didn't. Interestingly it shows wrong count when I am logged in, when logged out it  shows 1. And I don't have any private repos.

**Reply by NotchL1118** [reply]
> No I didn't. Interestingly it shows wrong count when I am logged in, when logged out it  shows 1. And I don't have any private repos.

My solution is to create a public repository, then delete the repository and return to normal

**Reply by gmoniava** [reply]
I would not like to delete the repository. Let's see maybe someone will spot and fix it.

**Reply by NotchL1118** [reply]
> I would not like to delete the repository. Let's see maybe someone will spot and fix it.

nono，u misunderstand me. create a new public repository and delete the new one，and the number will be right. trust me.

**Reply by gmoniava** [reply]
Ok, I will try later. But hopefully github fixes it.

**Reply by NotchL1118** [reply]
> Ok, I will try later. But hopefully github fixes it.

i find that someone else had the same question 4 years ago.LOL.Just do what I said.It may helps :)  Have a good Night or Day(Anyway.it is midnight in my country. good night)

**Reply by Tsyshiu** [reply]
I created a new repo, and deleted it, but the problem still exists😭

**Comment by NotchL1118** [comment]
It seems that this amount will refresh only if a public repository is created

**Comment by kaan35** [comment]
I have same issue

![image](https://user-images.githubusercontent.com/25592633/173821258-57dba75a-62df-4585-a7d3-975b4baaaab2.png)

**Reply by 5HT2** [reply]
No you don't. You're filtering to public repositories.

**Reply by kaan35** [reply]
I don't have 25 repositories. I have only 9 repositories but counter shows that i have 25 repo

**Reply by shaheerzeb** [reply]
The same case is with me, I think it shows the total number of repositories we have created till date.

**Comment by shaheerzeb** [comment]
The same case is with me, I think it shows the total number of repositories we have created till date.

**Comment by jfullstackdev** [comment]
maybe caching issues, what you can do is to create a repo first then delete it, because it will refresh the count of the repo, maybe

then say after 30 minutes, log in in Incognito mode and see, 

upon checking one of your profiles too, it seems they are fixed by now

**Reply by gmoniava** [reply]
It got fixed by itself, now shows 1 repo
